cli/cmd/up/up.go

```diff
@@ -185,3 +214,20 @@ func getCurrentDir() string {
 	dir := filepath.Base(workingDir)
 	return strings.ToLower(dir)
 }
+
+func setObjLabels(objs []runtime.Object, s *riov1.Stack) ([]runtime.Object, error) {
+
+	for _, obj := range objs {
+
+		obj := obj.(metav1.Object)
+		labels := obj.GetLabels()
+		if labels == nil {
+			labels = make(map[string]string)
+		}
+		// match debugID of stack
+		labels[stackLabel] = s.Namespace + "/" + s.Name
```

Comment:
Don't have to add s.Namespace. Stack.Namespace can be determined by child.Namespace.
It could be 
`labels[stackLabel] =  s.Name`

Reply:
Also I think we need to do
` obj.SetLabels(labels)`
Maybe I am wrong, not sure if labels are actually saved.